audit/projects/kubernetes-public/secrets/k8s-infra-ci-robot-github-account-password/description.json

```diff
@@ -0,0 +1,7 @@
+{
+  "createTime": "2021-07-28T16:51:19.454161Z",
+  "name": "projects/127754664067/secrets/k8s-infra-ci-robot-github-account-password",
```

Comment:
/cc @ameukam 
Thank you for creating this, we should add this to ensure-main-project.sh